code/API_definitions/Edge-Application-Management.yaml

```diff
@@ -925,6 +931,38 @@ components:
         - unknown
       default: unknown
 
+    EdgeCloudZoneFlavors:
+      description: List of unique Name IDs of Infrastructure Flavors
```

Comment:
How should a flavor be visualized from a developer point of view. Is a flavor represent a virtual machine (VM) or a server node with given set of resources mapped to it?

Reply:
If say ir represent a single node then gpuMemory attribute alone may not be sufficient to allocate such a resource. There may be attributes like gpuCount, gpuFamily etc are also to be considered to meet the workload requirements which need GPU.

Reply:
Hi @gunjald, given that flavors introduce complexity to the API, I'll implement a change that removes them. This will provide greater flexibility for operators to allocate workloads of any size.